nginx-controller/nginx/configurator.go

```diff
@@ -157,19 +221,21 @@ func (cnf *Configurator) generateNginxCfg(ingEx *IngressEx, pems map[string]stri
 		var locations []Location
 		rootLocation := false
 
-		for _, path := range rule.HTTP.Paths {
-			upsName := getNameForUpstream(ingEx.Ingress, rule.Host, path.Backend.ServiceName)
+		if ingEx.Ingress.Annotations["nginx.org/mergible-ingress-type"] != "master" {
```

Comment:
this check is not required.

Reply:
again, can't see why this check is required

Reply:
Removed.